src/ast/mod.rs

```diff
@@ -829,7 +832,19 @@ impl fmt::Display for Statement {
             Statement::Rollback { chain } => {
                 write!(f, "ROLLBACK{}", if *chain { " AND CHAIN" } else { "" },)
             }
-            Statement::CreateSchema { schema_name } => write!(f, "CREATE SCHEMA {}", schema_name),
+            Statement::CreateSchema {
+                schema_name,
+                if_not_exists,
+            } => write!(
+                f,
+                "CREATE SCHEMA{if_not_exists}{name}",
```

Comment:
Using `"CREATE SCHEMA {if_not_exists}{name}"` and an empty string for `if_not_exists` when it needs to be omitted both is consistent with other implementations, and results in more compact code.